src/marketplace/sellers/marketing-review-guidelines.md

```diff
@@ -9,32 +9,83 @@ From a marketing perspective, all submissions to Magento Marketplace must solve
 
 We have made an investment in the presentation of Magento Marketplace. In return, we ask that you put forth an effort to present your solutions in a professional manner. If we see room for improvement, we’ll reach out to you before listing your extension on Magento Marketplace.
 
--  Your product must have a professional presentation with clear descriptions, images, and screenshots.
--  The product name should indicate the purpose of the extension or theme.
--  The product description should be well-written, and provide an overview of the features and benefits.
--  Use only high-quality screenshots, images, and video to accompany the listing.
+*  Your product must have a professional presentation with clear descriptions, images, and screenshots.
+*  The product name should indicate the purpose of the extension or theme.
+*  The product description should be well-written, and provide an overview of the features and benefits.
+*  Use only high-quality screenshots, images, and video to accompany the listing.
+
+## Extension title
+
+The extension title should fulfill the following guidelines:
+
+*  Choose a title that is descriptive and concise. The extension title can be up to five words long and must be in English.
+*  Do not include the version number, developer name, or any reference to Magento in the title.
+   *  Do not use _"Magento"_ or _"M2"_.
+*  Do not include the words _"Extension"_, _"Module"_ or _"Plugin"_.
```

Comment:
I would suggest in this and the following bulleted items that you either use italics or quotations (I suggest italics) and not both.

Reply:
Please check all bulleted items for formatting and complete sentences.

Reply:
You didn't apply changes per my comment above, so I've done the work for you and added the suggestions for you to commit.

I will be requesting a review from @dobooth as well before merging.